apps-rendering/src/atoms.ts

```diff
@@ -302,6 +302,8 @@ function parseAtom(
 				return {
 					text: question.questionText,
 					...question,
+					/* TODO: imageUrl & imageAlt should be set here,
```

Comment:
Worth creating an GitHub issue for this?

Reply:
Good idea! I think we can track it in #6003